README.md

```diff
@@ -0,0 +1,208 @@
+
+## Table of contents 
+  - [A. Preparation](#a-preparation)
+    - [1. Kobiton username and API key](#1-kobiton-username-and-api-key)
+    - [2. Sample](#2-sample)
+  - [B. Setup](#b-setup)
+    - [1. Setting environment variables](#1-setting-environment-variables)
+    - [2. Getting desired capabilities](#2-getting-desired-capabilities)
+    - [3. Configuring automation test script](#3-configuring-automation-test-script)
+    - [4. Configuring Travis CI](#4-configuring-travis-ci)
+  - [C. Execution](#c-execution)
+    - [1. Run automation script on Kobiton devices](#1-run-automation-script-on-kobiton-devices)
+    - [2. Fetch session data through REST API](#2-fetch-session-data-through-rest-api)
+  - [D. Feedback](#d-feedback)
+
+## A. Preparation
+
+## 1. Kobiton username and API key
+
+In order to execute automation test on Kobiton devices, you need to prepare your Kobiton username and API key for authenticating with Kobiton. 
+  
+> Please visit https://portal.kobiton.com/register to create a new Kobiton account if you do not have one.
+
+Follow instructions at `IV. Configure Test Scripts for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) to get Kobiton username and API key.
+
+## 2. Sample
+
+We have provided a sample for automation test with Travis CI configuration file in this repository.
+
+The sample includes:
+- Basic configuration for Travis CI (`.travis.yml`)
+- Simple automation test script written in NodeJs. (`samples/automation-script` folder)
+
+> After this guideline, you can make your own adjustment to meet your requirements.
+
+To use the provided sample, follow these steps:
+
+1. Fork this repository https://github.com/kobiton/Travis-HockeyApp-Appium-ReactNative/tree/master
+2. Synchronize the forked repository with Travis CI. Visit [Travis CI document](https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/#how-to-migrate-a-private-repository-to-github-apps) for detailed instruction.
+3. Clone the forked repository.
+
+The initial build on Travis CI will fail as we haven't done any configuration. Please follow instructions in the next parts to configure for automation testing.
+
+## B. Setup
+
+This part will guide you how to set your Kobiton username and API key as environment variables and apply Kobiton device's desired capabilities to your automation test script. After that, we will configure Travis CI to execute the test on your preferred platform (Android/ iOS)
+
+## 1. Setting environment variables
+
+  Go to your Travis CI build project settings.
+
+  Add these two environment variable:
+  
+  -  KOBITON_USERNAME : Your Kobiton's username.
+  -  KOBITON_API_KEY : Your Kobiton's API Key.
+  
+  Please make sure to hide `KOBITON_API_KEY` value so it won't be exposed in build log.
+  
+  The result should look like below:
+  
+  ![](docs/assets/travis_env.jpg)
+
+
+## 2. Getting desired capabilities
```

Comment:
I don't know why do you need to add two case in this part, this really make me confuse. Instead you can do somethings like this: 

```
The desired capabilities need to be added to the automation test script in order for tests to be executed on the Kobiton device.

Note: The desiredCap are just matching only for one devices and in this guide we will use Pixel 2 XL - ver 8.1.0 (Android devices)  as demonstration. If you want to test on other devices, please make sure noted the different and customize as your need.  
```